services/housetables/src/main/java/com/linkedin/openhouse/housetables/dto/mapper/UserTablesMapper.java

```diff
@@ -26,19 +26,24 @@
     uses = {UserTableVersionMapper.class})
 public interface UserTablesMapper {
 
+  String DEFAULT_STORAGE = "hdfs";
+
   /**
    * From a source {@link UserTable}, prepare a {@link UserTableRow} object which is compliant with
    * the storage {@link
    * com.linkedin.openhouse.housetables.repository.impl.iceberg.UserTableHtsRepository},
    * particularly wrt its {@link Long} version requirements.
    *
+   * <p>Also sets the default storageType as "hdfs" if not present.
+   *
    * @param userTable userTable source to be mapped
    * @param existingUserTableRow if not present, set "version" as 1. If present, set "version" as
    *     the last saved version as long as: existingUserTableRow's metadata location and userTable's
    *     version match.
    * @return Destination House Table Data Model for storing Table Metadata.
    */
   @Mapping(target = "version", source = "userTable", qualifiedByName = "toVersion")
+  @Mapping(target = "storageType", source = "userTable.storageType", defaultValue = DEFAULT_STORAGE)
   UserTableRow toUserTableRow(
```

Comment:
### null -> 'hdfs' mapping happens here!

Another option was to do this by setting default in `api/spec/model/UserTable.java` using annotations such as `@DefaultValue`. But this wouldn't work cleanly with `/hts/tables/query` query, where null has a special semantics (null means `*` in that search api, which would conflict with put where null would mean `hdfs`)